### PR TITLE
Bump databroker to 2.0.0b2 and tiled to v0.1.0a60

### PR DESCRIPTION
**Pull request by danielballan**

### CODE REVIEW

**Comment by danielballan** [comment]
Bumping from Python 3.7 to 3.9 as well

**Review by mrakitin — APPROVED** [review]
Looks consistent with the packages deployed on the floor recently.

File: binder/postBuild [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-git clone --single-branch --branch v0.1.0a58 https://github.com/bluesky/tiled.git /tmp/tiled
-git clone --single-branch --branch v2.0.0a24 https://github.com/bluesky/databroker.git /tmp/databroker
+git clone --single-branch --branch v0.1.0a60 https://github.com/bluesky/tiled.git /tmp/tiled
+git clone --single-branch --branch v2.0.0b2 https://github.com/bluesky/databroker.git /tmp/databroker
```
Comment (mrakitin) [inline binder/postBuild:4]: As we already have the v2.0.0b2 in `conda-forge`, this repo may not be needed here anymore. Here is what was downloaded in the CI run:
```
  + databroker                                    2.0.0b2  pyhd8ed1ab_0            conda-forge/noarch        130 KB
```
Thoughts?

Comment (danielballan) [reply binder/postBuild]: Good idea. Done in [86ce98f](https://github.com/NSLS-II/scipy-binder/pull/42/commits/86ce98fa6a4407b166311ff98bf797f70155792e).

**Review by mrakitin — APPROVED** [review]
Ready to go.

**Pull request merged**